src/runtime/app.ts

```diff
@@ -83,7 +85,7 @@ export function create(): App {
   }
 
   const server = Server.create()
-  const schemaComponent = Schema.create()
+  const schemaComponent = Schema.create(__state)
```

Comment:
Interesting, I had being going a direction before where schema would have its own state. Your approach reminds me of a react-ish atomic-state idea. Any particular thoughts here or you just did what first came to you?